pkg/cluster/nodeutils/util.go

```diff
@@ -126,3 +126,41 @@ func ImageID(n nodes.Node, image string) (string, error) {
 	}
 	return crictlOut.Status.ID, nil
 }
+
+// ImageTags is used to perform a reverse lookup of the ImageID to list set of available
+// RepoTags corresponding to the ImageID in question
+func ImageTags(n nodes.Node, imageID string) (map[string]bool, error) {
+	var out bytes.Buffer
+	tags := make(map[string]bool, 0)
+	if err := n.Command("crictl", "inspecti", imageID).SetStdout(&out).Run(); err != nil {
+		return tags, err
+	}
+	crictlOut := struct {
+		Status struct {
+			RepoTags []string `json:"repoTags"`
+		} `json:"status"`
+	}{}
+	if err := json.Unmarshal(out.Bytes(), &crictlOut); err != nil {
+		return tags, err
+	}
+	for _, tag := range crictlOut.Status.RepoTags {
+		tags[tag] = true
+	}
+	return tags, nil
+}
+
+// ImageReTagSupported checks if the version of the ctr running in the cluster has support
+// for performing ctr images tag command
+func ImageReTagSupported(n nodes.Node) bool {
```

Comment:
I thought we were dropping this?

Reply:
Sorry, My bad. Yes we did. Forgot to cleanup. Let me take care of it.